x-pack/plugins/apm/public/components/shared/charts/failed_transaction_rate_chart/index.tsx

```diff
@@ -71,6 +73,14 @@ export function FailedTransactionRateChart({
 
   const { start, end } = useTimeRange({ rangeFrom, rangeTo });
 
+  const preferred = usePreferredDataSourceAndBucketSize({
+    start,
+    end,
+    numBuckets: 100,
+    kuery: '',
```

Comment:
Is this missing on purpose here?

Reply:
no, good catch - must have copied it from one of the alert details charts. thanks!